scripts/setup-centos9.sh

```diff
@@ -191,7 +191,7 @@ ARROW_VERSION=15.0.0
 
 function install_arrow {
   wget_and_untar https://archive.apache.org/dist/arrow/arrow-${ARROW_VERSION}/apache-arrow-${ARROW_VERSION}.tar.gz arrow
-  cd arrow/cpp
+  pushd arrow/cpp
```

Comment:
nit: The convention is to use a subshell by enclosing between `()`.
See `function install_duckdb` above.

Reply:
Yes, I see. Looks like the linked PR is fixing the issue. I will close this PR.